client/src/hooks/useOrganizationBests.js

```diff
@@ -75,6 +81,27 @@ export default function useOrganizationBests() {
         );
       }
 
+      if (filters.customizedOpenTimeFilter) {
+        // filter out inactive,inactiveTemporary stakeholders
+        filteredStakeholders = filteredStakeholders.filter((stakeholder) => {
```

Comment:
We can abstract the inactive & temporary inactive filter to a reusable helper function, other functionalities may be able to consume this filter.